singularity-eos/eos/eos_eospac.hpp

```diff
@@ -1208,6 +1235,44 @@ inline EOSPAC::EOSPAC(const int matid, bool invert_at_setup, Real insert_data,
       robust::ratio(dpde_ref_ * cv_ref_, rho_ref_ * rho_ref_ * pressureFromSesame(DPDR));
 }
 
+inline std::size_t EOSPAC::DynamicMemorySizeInBytes() const { return packed_size_; }
+inline std::size_t EOSPAC::DumpDynamicMemory(char *dst) {
+  static_assert(sizeof(char) == sizeof(EOS_CHAR), "EOS_CHAR is one byte");
+  EOS_INTEGER NTABLES[] = {NT};
+  EOS_INTEGER error_code = EOS_OK;
+  eos_GetPackedTables(NTABLES, tablehandle, (EOS_CHAR *)dst, &error_code);
+  eosCheckError(error_code, "eos_GetPackedTables", Verbosity::Debug);
+  return packed_size_; // JMM: Note this is NOT shared memory size
+}
+inline std::size_t EOSPAC::SetDynamicMemory(char *src, const SharedMemSettings &stngs) {
+  static_assert(sizeof(char) == sizeof(EOS_CHAR), "EOS_CHAR is one byte");
+  EOS_INTEGER NTABLES[] = {NT};
+  EOS_INTEGER error_code = EOS_OK;
+  eos_DestroyAll(&error_code);
```

Comment:
I think we can just destroy the table handles owned by the class. I _think_ the `Finalize()` member function should do this, right?

Reply:
Yes I think you're right. I will remove this `eos_DestroyAll` and just rely on the user calling `Finalize` appropriately as suggested by the documentation.